eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -210,22 +210,25 @@ public final Attr attr(final int pos) {
                 )
             );
         }
-        int idx;
-        for (idx = 0; idx < pos; ++idx) {
+        Attr attr = this.attr(this.order.get(0));
```

Comment:
@mximp Is it good way to assign "first" attribute here? What if  attribute with id `this.order.get(0)` doesn't exist?

Reply:
@volodya-lombrozo it always exists if all previous validations are passed.